demisto_sdk/commands/content_graph/strict_objects/base_strict_model.py

```diff
@@ -0,0 +1,117 @@
+from typing import Optional, Any, Union, Type
+
+import pydantic
+from pydantic.fields import FieldInfo
+
+from demisto_sdk.commands.common.StrEnum import StrEnum
+from pydantic import BaseModel, Extra, Field
+from demisto_sdk.commands.common.constants import Auto, MarketplaceVersions, TYPE_PWSH, TYPE_PYTHON, TYPE_JS
+
+marketplace_suffixes = [marketplace.value for marketplace in MarketplaceVersions]
+
+
+def create_dynamic_model(field_name: str, type_: Type, default: Any = ..., suffixes: list[str] = marketplace_suffixes,
+                         alias: Optional[str] = None):
+    """
+    This function creates a sub-model for avoiding duplicate lines of parsing arguments with different suffix.
+    Then the model inherit it.
+    """
```

Comment:
Let's add a very explicit example as it may be difficult for future maintainers to understand why we do that. 

1. Why we need it ("we have fields that are almost identical, except for the sufffix because <explain about the marketplaces>")
2. What it does ("it's equivalent to ... but better because")
3. Why the models below inherit from it (to add the fields to the root, as if they were declared in the strict models)